resume-buddy/src/main/java/com/google/sps/test/java/com/google/sps/ServletHelpersTest.java

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+@RunWith(JUnit4.class)
+public final class ServletHelpersTest {
+  /*TODO add tests for ServletHelpers class*/
+
+}
```

Comment:
This file should be in `resume-buddy/test/java/com/google/sps/ServletHelpersTest.java`, instead of `resume-buddy/src/main/java/com/google/sps/test/java/com/google/sps/ServletHelpersTest.java`

Reply:
Or you could just not have a test class yet, and put the todo somewhere in `ServletHelpers.java`